appengine/standard/firebase/firetactoe/firetactoe.py

```diff
@@ -53,20 +56,19 @@
 
 def _get_firebase_db_url(_memo={}):
     """Grabs the databaseURL from the Firebase config snippet."""
-    # Memoize the value, to avoid parsing the code snippet every time
+    # Memorize the value, to avoid parsing the code snippet every time
```

Comment:
Leave it then. Never used the term myself and it's got an edit distance of 1 from a statement that would also be true :)